src/sentry/templates/sentry/login.html

```diff
@@ -117,6 +125,35 @@ <h3>{% trans "Sign in to continue" %}</h3>
             </div>
           </form>
         </div>
+        {% if request.get_host == "pri-sentry.sentry.io" %}
+          {% if github_login_link or vsts_login_link or google_login_link %}
+            <div class="auth-provider-column">
+            {% if google_login_link %}
+              <p>
+                <a class="btn btn-default btn-login-google" href="{{ google_login_link }}" style="display: block">
+                  <span class="provider-logo google"></span> Sign up with Google
+                </a>
+              </p>
+            {% endif %}
+
+            {% if github_login_link %}
+              <p>
+                <a class="btn btn-default btn-login-github" href="{{ github_login_link }}" style="display: block">
+                  <span class="provider-logo github"></span> Sign up with GitHub
+                </a>
+              </p>
+            {% endif %}
+
+            {% if vsts_login_link %}
+              <p>
+                <a class="btn btn-default btn-login-vsts" href="{{ vsts_login_link }}" style="display: block">
+                  <span class="provider-logo vsts"></span> Sign up with Azure DevOps
+                </a>
+              </p>
+            {% endif %}
+          </div>
+          {% endif %}
+        {% endif %}
```

Comment:
Based on what I see on [our website](https://sentry.io/signup/), these links are the same to the ones we render in the login form. They are only going to have different `referrer ` when I merge the [PR](https://github.com/getsentry/getsentry/pull/16224)